### PR TITLE
feat: add feedback button

### LINKED ISSUE #525: Feature: Add feedback button in student UI ?
### Related user(s):

_No response_

### Related issue(s) or PR(s):

_No response_

### Related project scope(s):

web-app UI

### The problem:

In case of bug, many users won't have the habit to open the upstream repo and file an issue.

### Wanted solution:

* Add Feedback button in Learn-OCaml Web UI, linking to any of (or both):
  * https://github.com/learn-ocaml-community/teaching-feedback/
  * https://github.com/ocaml-sf/learn-ocaml/issues/new/choose

### Considered alternatives:

_No response_

### Additional context:

_No response_

### PR DESCRIPTION
**Pull request by Plictox**
* **Kind:** enhancement

* Close #525 

### Checklist

* [x] Read the [CONTRIBUTING.md](https://github.com/ocaml-sf/learn-ocaml/blob/master/CONTRIBUTING.md) guide and:
  * [x] Use [Atomic Commits](https://github.com/ocaml-sf/learn-ocaml/blob/master/CONTRIBUTING.md#atomic-commits) so each commit gathers a single logical change
  * [x] Use [Conventional Commits](https://github.com/ocaml-sf/learn-ocaml/blob/master/CONTRIBUTING.md#conventional-commits) regarding commit messages (needed by our release toolchain)
* [ ] Add/update [tests](https://github.com/ocaml-sf/learn-ocaml/tree/master/tests#readme)
  
* [ ] Add/update [documentation](https://github.com/ocaml-sf/learn-ocaml/tree/master/docs)
  
* [ ] …
  

### Note to maintainers

* Read [this wiki page](https://github.com/ocaml-sf/learn-ocaml/wiki/Checklist-for-testing-and-merging-a-PR).
* Make sure the PR has a **milestone**.
* ***Assign*** yourself before merging.
* Either do a regular **merge**:
  * for PRs containing *several commits* following [conventional-commits](https://github.com/ocaml-sf/learn-ocaml/blob/master/CONTRIBUTING.md#conventional-commits),
  * or for PRs containing 1 commit shared with a later PR (to preserve the SHA1)
* Or do a **squash-merge**:
  * for PRs containing *only 1 commit* (not shared with a later PR),
  * or for PRs containing several commits that need not be kept in the history;
  * → ***Update the commit message header*** with a [conventional-commit type](https://github.com/ocaml-sf/learn-ocaml/blob/master/CONTRIBUTING.md#conventional-commits-examples),
  * → ***Add a footer*** `Close #…` if a related issue exists.

### CODE REVIEW

File: static/css/learnocaml_main.css [hunk 1]
```diff
@@ -224,7 +229,7 @@ body {
     flex-direction: row;
   }
   #learnocaml-main-toolbar > button {
-    border-left: 1px #eee solid !important;
+    border-right: 1px #eee solid !important;
```
Comment (erikmd) [inline static/css/learnocaml_main.css:232]: Is it really needed?
```suggestion
    border-left: 1px #eee solid !important;
```

File: static/index.html [hunk 2]
```diff
@@ -25,6 +25,10 @@
       <div class="logo">
         <img src="/icons/logo_ocsf.svg">
         <span>Learn OCaml</span>
+        <button id ="feedback" onclick="window.location.href='https://github.com/ocaml-sf/learn-ocaml/issues/new/choose';" target = "_blank">
+          <!-- <span class = "label"> Feedback </span> -->
+          <img src="/icons/icon_feedback.svg" style="width: 22px">
+        </button>
```
Comment (erikmd) [inline static/index.html:31]:
```suggestion
        <a class="btn" href="https://github.com/learn-ocaml-community/teaching-feedback#readme" target = "_blank">
          <img src="/icons/icon_feedback.svg">
```

(to comply with usual *accessibility rules*: `<a>` is better than `<button>` here, https://a11y-101.com/design/button-vs-link )

**and TODO:** You forgot to commit the .svg image.

**Review by erikmd — CHANGES_REQUESTED** [review]
Thanks @Plictox !

I left a few comments for your PR

**Comment by erikmd** [comment]
> Suggestion (to tweak further w.r.t. spacing)

actually the spacing looks fine now.

Two leftover tasks:

* Convert draw style from black to white
* Add an id `` to make it possible to add an attribute `title="Send Feedback"`, internationalized, thanks to some js_of_ocaml code.

**Review by erikmd — APPROVED** [review]
@Plictox, that looks very good to me, thanks!

Here is the outcome of your PR:

2023-04-18_19-12-04_Screenshot_a_img

→ note in passing that your CSS code induced a slight (unwanted) shift of the `Learn OCaml` text. This is only due to the `margin-top:` property, that is unnecessary.

Finally, note that the `margin-left` should rather be put outside the link `<a href=_>` rather than inside, so that the surrounding black space is not clickable.

I'll push a commit and rebase your pull request, so that we can get two final commits to be merged.

Thanks.

**Comment by erikmd** [comment]
Now, let's wait for the CI :white_check_mark:

**Comment by Plictox** [comment]
This feature could be improved by implementing a sidebar. It would be more complex but way more elegant ! cf the google feedback option
image

**Pull request merged**